### PR TITLE
Integrate cisagov/cool-cdm-cloudtrail-tf-module

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request integrates [cisagov/cool-cdm-cloudtrail-tf-module](https://github.com/cisagov/cool-cdm-cloudtrail-tf-module) into this repository.

## 💭 Motivation and context ##

This integration allows CDM's Splunk to ingest the CloudTrail logs that are generated from this AWS account.

## 🧪 Testing ##

I have applied these changes to our production COOL SharedServices account, and the CDM folks tell me that they are successfully ingesting CloudTrail logs.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+output "cdm_cloudtrail_topic" {
+  value       = module.cdm_cloudtrail.topic
+  description = "The SNS topic for notifications of CloudTrail logs being written to the CDM S3 bucket."
+}
+
+output "cdm_cloudtrail_trail" {
+  value       = module.cdm_cloudtrail.trail
+  description = "The CloudTrail trail for CDM."
+}
+
```
Comment (dav3r) [inline outputs.tf:40]: Might it be simpler here to just make a single output for the entire `module.cdm_cloudtrail`?  Or is there something in there that you don't want to expose here?  It is nice having all of these descriptions here, so I'm fine either way.  Just tossing it out there.

**Review by dav3r — APPROVED** [review]
Excelsior!  💥 
I had one question/comment, but it doesn't affect approval.

File: outputs.tf (hunk 1 above)
Comment (jsf9k) [reply outputs.tf:40]: I thought about that, but then the user has to go look up the outputs of the module to know what is available.  I thought this was a more useful solution for anyone who is dipping into the remote state to grab these values.

**Review by mcdonnnj — APPROVED** [review]
Makes sense to me. Strong work with all of this CDM integration 💪💪💪

**Pull request merged**